pkg/scheduler/framework/plugins/serviceaffinity/service_affinity.go

```diff
@@ -108,14 +108,14 @@ func (pl *ServiceAffinity) createPreFilterState(pod *v1.Pod) (*preFilterState, e
 	// Store services which match the pod.
 	matchingPodServices, err := helper.GetPodServices(pl.serviceLister, pod)
 	if err != nil {
-		return nil, fmt.Errorf("listing pod services: %v", err.Error())
+		return nil, fmt.Errorf("listing pod services: %w", err.Error())
```

Comment:
oh, don't call `err.Error()`, because that's a string

Reply:
I was too careless to notice the problem. Thank you very much. It has been revised.